tests/test_invessel_build.py

```diff
@@ -94,8 +96,10 @@ def test_ivb_basics(invessel_build):
     remove_files()
 
 
-def test_ivb_construction(invessel_build):
-
+def test_ivb_cadquery_construction(invessel_build):
+    """Tests whether the InVesselBuild CadQuery workflow functions as
+    expected.
```

Comment:
Might be good to include a little more detail, e.g.:
```suggestion
    expected, by testing whether:
       * the correct number of components were assembled
       * the first rib has the correct number of points
       * the rib is defined by floating point numbers
```